android/modules/media/src/java/ti/modules/titanium/media/MediaModule.java

```diff
@@ -867,6 +869,14 @@ private static void ensureExternalPublicMediaDirectoryExists()
 		}
 	}
 
+	@Kroll.method
+	public String harmonizerColor(String value)
```

Comment:
I would rather see this under `Ti.UI` or `Ti.UI.Android`, what do you think? Also, I would prefer `harmonizedColor`

Reply:
both valid points :+1: will move it